dataland-frontend/src/components/pages/SearchCompaniesForFrameworkData.vue

```diff
@@ -90,15 +118,28 @@ export default defineComponent({
       latestScrollPosition: 0,
       currentSearchBarInput: "",
       currentFilteredFrameworks: [] as Array<DataTypeEnum>,
```

Comment:
Das macht meines Erachtens keinen Sinn das Array leer zu lassen, weil er dann bei "scanQueryParams" (auch wenn gar keine query params angegeben sind) in getQueryParams reingeht und die currentQueryFrameworks updatet

Reply:
Ergebnis aus Gespräch: Man könnte auch direkt alle frameworks reintun.  Probier es einfach mal aus und wenns dann klappt und er nicht mehr in getQueryParams reingeht passt es.